src/app/components/forms/user/password-reset/password-reset.component.html

```diff
@@ -0,0 +1,68 @@
+<h1>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_TITLE | translate
+  }}
+</h1>
+<h2>
+  {{
+    i18nService.i18nComponentsEnum.RESET_PASSWORD_COMPONENT_SUBTITLE | translate
+  }}
+</h2>
+
+<form
+  [formGroup]="passwordResetForm"
+  (ngSubmit)="onSubmit()"
+  autocomplete="off"
+  class="password-reset-form"
+>
+  <mat-form-field appearance="outline" class="full-width">
+    <mat-label>{{
+      i18nService.i18nFormsEnum.CURRENT_PASSWORD_LABEL | translate
+    }}</mat-label>
+    <input matInput formControlName="oldPassword" type="password" required />
+    <mat-error *ngIf="submitted() && oldPassword.hasError('required')">
+      {{ i18nService.I18nFormsEnum.CURRENT_PASSWORD_LABEL | translate }}
+    </mat-error>
+  </mat-form-field>
+
+  <mat-form-field appearance="outline" class="full-width">
+    <mat-label>{{
+      i18nService.i18nFormsEnum.NEW_PASSWORD_LABEL | translate
+    }}</mat-label>
+    <input matInput formControlName="newPassword" type="password" required />
+    <mat-error *ngIf="submitted() && newPassword.hasError('required')">
+      {{ i18nService.I18nFormsEnum.FORM_VALIDATION_REQUIRED | translate }}
+    </mat-error>
+    <mat-error *ngIf="submitted() && newPassword.hasError('minlength')">
+      {{ "NEW_PASSWORD_MIN_LENGTH" | translate : { min: 6 } }}
+    </mat-error>
```

Comment:
este..... ummmm tendria que revisarlo porque le estas parando un parametro entonces este mejor no lo cambies. nada mas te voy a pasar como seria con la sintaxis nueva.

```suggestion
@let newPasswordControl = form().get('newPassword');
@if(submitted() && newPassword.hasError('minlength')){
        {{ "NEW_PASSWORD_MIN_LENGTH" | translate : { min: 6 } }}
}
```

Cambie recalcar que este 6 no deberia estar quemado pero dejemolo ahi de momento (esperemos que olman no lo vea y nos baje puntos jaja.